docs/Examples/Assets.md

```diff
@@ -26,15 +26,15 @@ enum Asset: TargetType {
   }
 
   var localLocation: URL {
```

Comment:
how about moving these to one-liners as well? (if it makes sense ofc)

Reply:
Done!